contrib/flo-scio_2.11/pom.xml

```diff
@@ -34,6 +34,7 @@
       <groupId>com.spotify</groupId>
       <artifactId>scio-test_2.11</artifactId>
       <version>0.5.5</version>
```

Comment:
Not part of this PR, but was it on purpose we don't put `version` in parent pom `dependencyManagement`?

Reply:
Probably because that's a scala 2.11 specific dependency.

Reply:
For this https://github.com/spotify/flo/pull/168/files#diff-9b2c165d188171b45f6d1e1dcff5b3f5R26, I usually prefer putting it in parent as well, such as https://github.com/spotify/flo/blob/617987f92362d68baaef38b8d0ec831949458b5d/pom.xml#L60

Reply:
Compare with https://github.com/spotify/flo/blob/master/flo-scala_2.11/pom.xml#L48